src/components/drawer-item/drawer-item.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, h, Host, Prop } from "@stencil/core"
+
+@Component({
+  tag: "oui-drawer-item",
+  styleUrl: "./drawer-item.css",
+})
+export class DrawerItem {
+  /**
+   * Set drawer item link if string will be treated as a url or as a callback if it is a function
+   */
+  @Prop() public link: string
```

Comment:
if this property can be a function it should be type hinted and tested, also I would prefer removing the link option and just implementing an `onClick`